pwdtk/apps.py

```diff
@@ -24,6 +24,14 @@ def ready(self):
             logger.debug("PWDTK DISABLED")
             return
 
-        User = get_user_model()
-        from pwdtk.watchers import watch_set_password
-        User.set_password = watch_set_password(User.set_password)
+        if not hasattr(settings, "AUTH_PASSWORD_VALIDATORS"):
+            settings.AUTH_PASSWORD_VALIDATORS = []
```

Comment:
is it really good practice to modify django settings?

Reply:
i remove but we still add PWDTK_PASSWORD_VALIDATOR to AUTH_PASSWORD_VALIDATORS